src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -599,6 +600,23 @@ public void readPath(
         isFabricFiltered);
   }
 
+  /** Read the given attribute/event path with isFabricFiltered flag. */
```

Comment:
This documentation seems incorrect -- it would be good to annotate the `@param` tags to describe the inputs too.

Reply:
done, thanks